apps/openmw/mwmechanics/mechanicsmanagerimp.cpp

```diff
@@ -1094,6 +1082,7 @@ namespace MWMechanics
         }
 
         // Make surrounding actors within alarm distance respond to the crime
+        // TODO: Vanilla doesn't make neighbors attack player if one of his follower/summon attacks first (probably vanilla bug)
```

Comment:
Yeah that sounds like the right vanilla behavior. When getting boots of blinding speed, the woman I was helping would just zip off to kill enemies which was annoying to say the least...